### PR TITLE
MVP-2364: WalletConnect support in SDK

### PR DESCRIPTION
**Pull request by eric-notifi**
# WalletConnect supported in SDK (Doc + example code).

1. Add WalletConnect example to notifi-react-example / Classify supported Blockchains (wallets) by switchable view.
2. Update the README.md of notifi-react-card to showcase WalletConnect.
3. Update the README.md of notifi-react-example

https://user-images.githubusercontent.com/127958634/229663169-c51575d5-6ec7-411c-9443-c991620a59df.mov

### CODE REVIEW

File: packages/notifi-react-card/README.md [hunk 1]
```diff
@@ -590,3 +590,103 @@ export const Notifi: React.FC = () => {
...
+  const client = createClient({
+    autoConnect: true,
+    connectors: [connector],
+    provider: provider,
+  });
+  return <WagmiConfig client={client}>{children}</WagmiConfig>;
+};
+```
+
+Wrap the React </App> by <WalletConnectProvider />
```
Comment (Nathan-Notifi) [inline packages/notifi-react-card/README.md:633]:
```suggestion
Wrap the </App> with <WalletConnectProvider />
```

File: packages/notifi-react-card/README.md [hunk 2]
```diff
@@ -590,3 +590,103 @@ export const Notifi: React.FC = () => {
...
+```tsx
+import { FC, PropsWithChildren } from 'react';
+import { WagmiConfig, configureChains, createClient, mainnet } from 'wagmi';
+import { WalletConnectConnector } from 'wagmi/connectors/walletConnect';
+import { infuraProvider } from 'wagmi/providers/infura';
+
+export const connector = new WalletConnectConnector({
+  chains: [mainnet],
+  options: {
+    projectId: '<YOUR WALLETCONNECTOR PROJECT ID HERE>', // Get Project ID at https://cloud.walletconnect.com/
```
Comment (Nathan-Notifi) [inline packages/notifi-react-card/README.md:615]:
```suggestion
    projectId: '<YOUR WALLETCONNECTOR PROJECT ID HERE>', // Get Project ID at https://cloud.walletconnect.com/
```

Should it be?

    projectId: '<YOUR WALLETCONNECT PROJECT ID HERE>', // Get Project ID at https://cloud.walletconnect.com/

File: packages/notifi-react-example/src/App.tsx [hunk 3]
```diff
@@ -2,11 +2,13 @@ import React from 'react';
 
 import './App.css';
 import { NotifiCard } from './NotifiCard/NotifiCard';
+import { WalletConnectCard } from './NotifiCard/WalletConnectCard';
 
 function App() {
   return (
     <div className="App">
       <NotifiCard />
```
Comment (Nathan-Notifi) [inline packages/notifi-react-example/src/App.tsx:10]: I know in our app we have a few different cards, could we limit them through a parameter we pass down?

Or maybe we can do a toggle, I think there are several requests being sent and using a walletConnect might be adding some more.

I know we have the multi wallet too, maybe we can separate them into different views

File: packages/notifi-react-example/src/WalletConnectProvider.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    [mainnet],
+    [infuraProvider({ apiKey: '9c9ff698105d4f6b9b2b93eddc0dff72' })],
+  );
+  const client = createClient({
+    autoConnect: true,
+    connectors: [connector],
+    provider: provider,
+  });
+  return (
+    <>
```
Comment (Nathan-Notifi) [inline packages/notifi-react-example/src/WalletConnectProvider.tsx:25]: oo, do we need this fragment?

**Review by juni-notifi — APPROVED** [review]

**Pull request merged**